pkgs/development/python-modules/enochecker_test/default.nix

```diff
@@ -0,0 +1,65 @@
+{ lib
+, buildPythonPackage
+, fetchPypi
+, pythonOlder
+, pythonRelaxDepsHook
+
+, certifi
+, charset-normalizer
+, enochecker_core
+, exceptiongroup
+, idna
+, iniconfig
+, jsons
+, packaging
+, pluggy
+, pytest
+, requests
+, tomli
+, typish
+, urllib3
+}:
+
+buildPythonPackage rec {
+  pname = "enochecker_test";
+  version = "0.9.0";
+
+  disabled = pythonOlder "3.8";
+
+  src = fetchPypi {
+    inherit pname version;
+    hash = "sha256-M0RTstFePU7O51YVEncVDuuR6F7R8mfdKbO0j7k/o8Q=";
+  };
+
+  nativeBuildInputs = [
+    pythonRelaxDepsHook
+  ];
+
+  pythonRelaxDeps = true;
+
+  propagatedBuildInputs = [
+    certifi
+    charset-normalizer
+    enochecker_core
+    exceptiongroup
+    idna
+    iniconfig
+    jsons
+    packaging
+    pluggy
+    pytest
+    requests
+    tomli
+    typish
+    urllib3
+  ];
+
+  doCheck = false;
```

Comment:
Why disable the tests?

Reply:
Anyway, `pythonImportsCheck` is needed.

Reply:
> Why disable the tests?

Testing fails with the following [log](https://github.com/NixOS/nixpkgs/files/11748713/enochecker_test-tests.log). As far as I understand, not the test itself fails but during test setup, the test runner / setuptools tries to download the `requirements.txt` packages.

I do not know how to debug/fix this (pointers much appreciated), thus I disabled them for now.

Reply:
> Anyway, `pythonImportsCheck` is needed.

Is it even when the package isn't really meant to be imported since it mainly contains a to-be executed script?

Reply:
So I'd move the folder with the `default.nix` into e.g. `/pkgs/development/tools/` and move the `enochecker_test` decl to `all-packages.nix`?

Or should I simply add a toplevel entry like [this one](https://github.com/NixOS/nixpkgs/blob/nixos-23.05/pkgs/top-level/all-packages.nix#L8586)?

Reply:
Since it is not expected to be used as a library, it is best to do the former.

Reply:
done. Thanks alot for the quick, nice feedback :)